### PR TITLE
Versioning generated files for crate proto

### PR DESCRIPTION
**Pull request by overvenus**

### CODE REVIEW

File: proto/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "grpcio-proto"
-version = "0.1.2"
+version = "0.1.3"
```
Comment (BusyJay) [inline proto/Cargo.toml]: Don't update it now.

File: src/test_grpc.rs [hunk 2]
```diff
@@ -0,0 +1,364 @@
+// This file is generated. Do not edit
```
Comment (BusyJay) [inline src/test_grpc.rs]: Why is the file here?

**Comment by siddontang** [comment]
what is the purpose of this PR?

File: proto/build.sh [hunk 3]
```diff
@@ -14,13 +26,68 @@ if [[ $major -lt 3 ]]; then
...
 	include="$rootdir/proto/$include"
-	echo building $name.desc
-	find "$prefix" -name "*.proto" | xargs protoc -I "$include" -o "$rootdir/$name.desc"
+	output="$rootdir/src/$package"
+	echo building package $package
+	mkdir -p $output
+	find "$prefix" -name "*.proto" | xargs protoc -I "$include" --rust_out $output
+	find "$prefix" -name "*.proto" | xargs protoc -I "$include" --grpc_out $output --plugin=protoc-gen-grpc=`which grpc_rust_plugin`
+	push $output
+	rm -f mod.rs
```
Comment (BusyJay) [inline proto/build.sh]: Why?

**Pull request closed without merging**